localstack/services/sqs/provider.py

```diff
@@ -152,6 +155,16 @@ def assert_queue_name(queue_name: str, fifo: bool = False):
         )
 
 
+def check_message_size(message_body: str, max_message_size: int):
+    # https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/quotas-messages.html
+    error = "InvalidParameterValue. One or more parameters are invalid. "
+    error += "Reason: Message must be shorter than 262144 bytes."
```

Comment:
judging from the snapshot, i think this is the correct message, but please double check:
```suggestion
    error = "One or more parameters are invalid. "
    error += f"Reason: Message must be shorter than {max_message_size} bytes."
```

Reply:
Thank you very much for you detailed and patient review. I will wirk through and incorporate these changes. I appreciate your time in explaining this.

Reply:
Working on this now.  Thank you for fixing the tests.  I will look more closely at the methods of testing.